data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/illuminates/illuminates_negative_min_value.cfg

```diff
@@ -0,0 +1,33 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [illuminates]min_value=
+##
+# Actions:
+# Give all units 50% self-illuminates bonus damage with a min_value=-10
```

Comment:
-50%

Reply:
This comment is correct, actually.

Reply:
I'm confused - there's a `-50` on the `TEST_ABILITY` line, and the test seems to be checking that `min_value=-10` brings that up to a total of 90% damage instead of a total of 50% damage.

Reply:
Oh, I misunderstood - I though you were saying it should say `min_value=-50`. Updated.